.github/workflows/test.yml

```diff
@@ -31,7 +31,7 @@ jobs:
           --health-timeout 5s
           --health-retries 5
       mssql:
-        image: mcr.microsoft.com/mssql/server:2019-latest
+        image: mcr.microsoft.com/mssql/server:2019-CU30-ubuntu-20.04
```

Comment:
Since the un-related tests is still failed, please revert this. We'll fix the tests in another PR

Reply:
ye, I reverted in c4e0cf6